### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by omercnet**
* error on invalid subcommand
* catch timeout exception on service test failure
* fix missing `config.json` error

### CODE REVIEW

**Comment by Sleeptime17** [comment]
Looks reasonable to me, just wanna ask why the travis-ci build failed before you merge this?

**Comment by omercnet** [comment]
@Sleeptime17 please take another loop and approve

File: tests/test_honeycomb.py [hunk 1]
```diff
@@ -438,63 +438,62 @@ def test_integration_run(running_service_with_integration, syslogd):
...
-        p = subprocess.Popen(RUN_HONEYCOMB + cmdargs, env=os.environ, stdout=subprocess.PIPE)
-        wait_for_output(p, "Starting Simple HTTP service")
-
-        rsession.get("http://localhost:{}/{}".format(DEMO_SERVICE_PORT, teststring))
-        wait_for_output(p, teststring)
-
-        p.send_signal(signal.SIGINT)
-        assert wait_until(p.wait)
-    """
+# @pytest.mark.dependency(depends=["service_daemon"])
```
Comment (Sleeptime17) [inline tests/test_honeycomb.py:441]: Why is this all commented out? If it's not useful, delete it; is it only semi-functional so you're keeping it for the future?

Comment (omercnet) [reply tests/test_honeycomb.py:441]: it's a working draft that I don't wanna lose

**Review by Sleeptime17 — APPROVED** [review]
I'm not happy about the commented functions but okay

File: tests/test_honeycomb.py (hunk 1 above)
Comment (omercnet) [reply tests/test_honeycomb.py:441]: sorry I got confused with the other repo

this test is failing for a reason I can't seem to understand.. `result.output` is empty when run with other tests, but if I run it alone it's fine
the fixtures are scoped to function by default so they shouldn't affect each other - I couldn't figure this out and wanted to move this along for now

**Pull request merged**